manim/mobject/coordinate_systems.py

```diff
@@ -310,6 +311,16 @@ def init_background_lines(self):
         )
 
     def get_lines(self):
+        """Generate all the lines, faded and not faded. Two sets of lines are generated, parallel to the X-axis and to Y-axis.
+        
+        Returns
+        -------
+        :class:`~.VGroup`
+            First set of lines (i.e the non faded lines)
+
+        :class:`~.VGroup`
+            Second set of lines (i.e the faded lines)
```

Comment:
A function may only have one return type. Note that the `a, b` syntax is syntactic sugar for `(a, b)` (returns a tuple).
```suggestion
        Tuple[:class:`~.VGroup`, :class:`~.VGroup`]
            The first (i.e the non faded lines) and second (i.e the faded lines) sets of lines, respectively.
```